src/main/java/com/minecolonies/coremod/network/messages/MobEntryChangeMessage.java

```diff
@@ -0,0 +1,93 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.api.colony.permissions.Action;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuardsNew;
+import com.minecolonies.coremod.colony.buildings.views.MobEntryView;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Creates a message for changing MobEntry priorities
+ */
+public class MobEntryChangeMessage extends AbstractMessage<MobEntryChangeMessage, IMessage>
+{
+    private int      colonyId;
+    private BlockPos buildingId;
+    private List<MobEntryView> mobsToAttack = new ArrayList<>();
+
+    /**
+     * Empty standard constructor.
+     */
+    public MobEntryChangeMessage()
+    {
+        super();
+    }
+
+    public MobEntryChangeMessage(
+      @NotNull final AbstractBuildingGuardsNew.View building,
+                                  final List<MobEntryView> mobsToAttack
+    )
+    {
+        super();
+        this.colonyId = building.getColony().getID();
+        this.buildingId = building.getID();
+        this.mobsToAttack = mobsToAttack;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "mobsToAttack". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)